include/searchresultsformaction.h

```diff
@@ -0,0 +1,43 @@
+#ifndef NEWSBOAT_SEARCHRESULTSFORMACTION_H_
+#define NEWSBOAT_SEARCHRESULTSFORMACTION_H_
+#include "configcontainer.h"
+#include "itemlistformaction.h"
+#include "logger.h"
+#include "regexmanager.h"
+#include "rssfeed.h"
+#include "view.h"
+#include "keymap.h"
+#include <stack>
+
+namespace newsboat {
+
+class SearchResultsFormAction : public ItemListFormAction {
+public:
+	SearchResultsFormAction(View* vv,
```

Comment:
It'd be nice if the constructor also took the initial feed to be added to the history, and also the searchphrase. That way, there's no need to call `set_feed()`, `add_history()`, and `set_searchphrase()` right after constructing the formaction, and then there's no way to screw things up by forgetting to call the methods.

Reply:
@Minoru but `ItemListFormAction` and other `FormAction` based classes don't do that. That consequence to inconsistent code and make a constructor too big.

Reply:
Okay, let's keep it then. I can always change it later if I feel strongly about it :)